ts/Dashboards/Component/Component.ts

```diff
@@ -1001,14 +1055,20 @@ namespace Component {
     }
 
     /**
+     * Extract all components names from the registry.
      *
+     * @return {Array<string>}
+     * Returns the array of components names.
      */
     export function getAllComponentNames(): Array<string> {
         return Object.keys(Component.registry);
     }
 
     /**
+     * Extract all components from the registry.
      *
+     * @return {Record<string, Class<Component>>}
+     * Returns the array of components.
      */
     export function getAllComponents(): Record<string, Class<Component>> {
```

Comment:
It isn't used anywhere. Do we need it?

Reply:
@goransle ?

Reply:
No opinion. It's meant to be used in with es6 imports and in a "dashboardless" setup